JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardTop.kt

```diff
@@ -48,39 +47,31 @@ import com.example.jetnews.ui.lightThemeColors
 @Composable
 fun PostCardTop(post: Post) {
 // TUTORIAL CONTENT STARTS HERE
-    val typography = MaterialTheme.typography()
-    Column(modifier = LayoutWidth.Fill + LayoutPadding(16.dp)) {
+    val typography = MaterialTheme.typography
+    Column(modifier = Modifier.fillMaxWidth().padding(16.dp)) {
```

Comment:
I starting to put one modifier per line. WDYT?

Reply:
If it's readable I tend towards that, in this case I think it hurts readability since it's on a Column and it drops the lambda start a few lines. I'd have to pull it out to a variable in order to multi-line it which didn't seem worth it for a simple application like this (if it grew another one I'd extract variable).

Picked this one after trying the options, but could also consider extracting a variable for this case.

Reply:
ack